backend/medtagger/database/models.py

```diff
@@ -71,6 +71,8 @@ class ScanCategory(Base):
     name: str = Column(String(100), nullable=False)
     image_path: str = Column(String(100), nullable=False)
 
+    tags: List['LabelTag'] = relationship("LabelTag", back_populates="scan_category")
```

Comment:
I would say that these tags should be named `available_tags`. What do you think?